CONTRIBUTING.md

```diff
@@ -0,0 +1,53 @@
+# 🔨👷 Contributing
+
+---
+
+## Table of Contents
+
+- [Dependencies](#dependencies)
+- [Getting Started](#getting-started)
+- [Submitting Issues](#submitting-issues)
+- [Changelog](#changelog)
+- [Queries](#queries)
+
+## Dependencies
```

Comment:
This should be in the ReadMe file under Getting Started

Reply:
Moved